lib/zendesk_apps_tools/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module ZendeskAppsTools
-  VERSION = '2.13.3'
+  VERSION = '2.13.4'
```

Comment:
You don't need to change the version here and in Gemfile.lock. We'll bump the version later when we release it.

Reply:
Reverted this change.